Lecture11/Tests.cs

```diff
@@ -73,6 +75,175 @@ public void OneTimeSetup()
+
+            var FirstNameTextBox = _driver.FindElement(By.Id("firstName"));   // finding elements of registration form
+            var LastNameTextBox = _driver.FindElement(By.Id("lastName"));
+            var EmailTextBox = _driver.FindElement(By.Id("userEmail"));
+            var AgeTextBox = _driver.FindElement(By.Id("age"));
+            var SalaryTextBox = _driver.FindElement(By.Id("salary"));
+            var DepartmentTextBox = _driver.FindElement(By.Id("department"));
+            var SubmitButton = _driver.FindElement(By.Id("submit"));
+
+            FirstNameTextBox.SendKeys("Walter");    // entering values to the registration form fields
+            LastNameTextBox.SendKeys("White");
+            EmailTextBox.SendKeys("heisenberg@gmail.com");
+            AgeTextBox.SendKeys("51");
+            SalaryTextBox.SendKeys("11000000");
+            DepartmentTextBox.SendKeys("Chemistry Department");
+            SubmitButton.Click();
+
+            var FirstNameTableColumn = _driver.FindElement(By.XPath("//div[@role=\"rowgroup\"][4]//div[@role=\"gridcell\"][1]"));   // finding columns of the fourth table row
+            var LastNameTableColumn = _driver.FindElement(By.XPath("//div[@role=\"rowgroup\"][4]//div[@role=\"gridcell\"][2]"));
+            var AgeTableColumn = _driver.FindElement(By.XPath("//div[@role=\"rowgroup\"][4]//div[@role=\"gridcell\"][3]"));
+            var EmailTableColumn = _driver.FindElement(By.XPath("//div[@role=\"rowgroup\"][4]//div[@role=\"gridcell\"][4]"));
+            var SalaryTableColumn = _driver.FindElement(By.XPath("//div[@role=\"rowgroup\"][4]//div[@role=\"gridcell\"][5]"));
+            var DepartmentTableColumn = _driver.FindElement(By.XPath("//div[@role=\"rowgroup\"][4]//div[@role=\"gridcell\"][6]"));
+
+            Assert.IsTrue(FirstNameTableColumn.Text.Equals("Walter"));         // asserts to verify that the entred data correctly displayed in the table
+            Assert.IsTrue(LastNameTableColumn.Text.Equals("White"));
+            Assert.IsTrue(AgeTableColumn.Text.Equals("51"));
+            Assert.IsTrue(EmailTableColumn.Text.Equals("heisenberg@gmail.com"));
+            Assert.IsTrue(SalaryTableColumn.Text.Equals("11000000"));
+            Assert.IsTrue(DepartmentTableColumn.Text.Equals("Chemistry Department"));
+        }
+
+        [Test]
+        public void ButtonsClickTest()     //test for Buttons
+        {
+            _driver.Navigate().GoToUrl("https://demoqa.com/buttons");         // opens the indicated url
+
+            var doubleClickButton = _driver.FindElement(By.Id("doubleClickBtn"));       // saving locators for the buttons
+            var rightClickButton = _driver.FindElement(By.Id("rightClickBtn"));
+            var clickButton = _driver.FindElement(By.XPath("//button[text()=\"Click Me\"]"));
+
+            _driverActions.DoubleClick(doubleClickButton).Perform();      // performing double click
```

Comment:
better to use Build().Perform()
fix everywhere

Reply:
done